packages/shared-metrics/test/dependencies/app-without-package-json/package.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
~~This is a bit confusing to me. The `app-without-package-json` has a package.json.
Is this intended or was maybe a copy and paste from the "with package.json" one?~~
Never mind. I saw the explanation in the test file.